contracts/interfaces/modules/grouping/IGroupRewardPool.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.23;
+
+/// @title IGroupingPolicy
+/// @notice Interface for grouping policies
+interface IGroupRewardPool {
```

Comment:
Why do we have this interface?

I'm thinking we may not need this interface for two reasons:
- no contract on the protocol calls it
- each pool can have a different interface

Correct me if I may be missing something - the only interaction between protocol contracts and external pools is just send ERC20 tokens - if so we don't need to call any function on those pools.